pkg/provision/source.go

```diff
@@ -65,6 +65,7 @@ func (s *pollSource) Reservations(ctx context.Context) <-chan *Reservation {
 			res, err := s.store.Poll(pkg.StrIdentifier(s.nodeID), next)
 			if err != nil && err != ErrPollEOS {
 				log.Error().Err(err).Msg("failed to get reservation")
+				next++
```

Comment:
Do we always want  to skip this reservation? I.E is not possible that poll returns a temporary network error or something

Reply:
I guess we could do that. but the reservation is never going to be fixed. What should be the behavior here if we get a "temporary error" ?

Reply:
Well, if its a network error where the explorer is temporarily unreachable (for this node), then it would be possible for this reservation to become valid in the future right? So skipping it here would mean that it would no longer get provisioned then

Reply:
Ok I see what you mean now. But we are talking about 2 different problem here.
- if the network is the problem, in this case we should indeed not increment next and just wait to retry
- the reservation format is invalid and we cannot unmarshal it. In this case the reservation should be skipped